app/controllers/concerns/billable_event_trackable.rb

```diff
@@ -33,7 +36,7 @@ def create_sp_return_log(billable:)
   end
 
   def current_session_has_been_billed?
-    user_session[session_has_been_billed_flag_key] == true
+    user_session[session_has_been_billed_flag_key]
```

Comment:
why did we change this part?

Reply:
No changes needed on this.Reverting this change.